Specs/Scene/Model/GeometryPipelineStageSpec.js

```diff
@@ -383,8 +383,8 @@ describe(
         expect(position2DAttribute.componentDatatype).toEqual(
           ComponentDatatype.FLOAT
         );
-        expect(position2DAttribute.offsetInBytes).toBe(288);
-        expect(position2DAttribute.strideInBytes).toBe(12);
+        expect(position2DAttribute.offsetInBytes).toBe(0);
```

Comment:
This test was wrong. The 2D positions are computed in a packed typed array before being put in a buffer. The reason this worked anyway is because `GltfLoader` was rewriting the `byteOffset` and `offsetInBytes` if loading only the `typedArray` and not the `buffer`, but this has been changed